resources/zib/zib-Procedure-ServiceRequest.xml

```diff
@@ -0,0 +1,219 @@
+      </binding>
+      <mapping>
+        <identity value="zib-procedure-v5.2-2020EN" />
+        <map value="NL-CM:14.1.4" />
+        <comment value="ProcedureType" />
+      </mapping>
+    </element>
+    <element id="ServiceRequest.orderDetail">
+      <path value="ServiceRequest.orderDetail" />
+      <slicing>
+        <discriminator>
+          <type value="value" />
+          <path value="$this" />
+        </discriminator>
+        <rules value="open" />
+      </slicing>
+    </element>
+    <element id="ServiceRequest.orderDetail:procedureMethod">
+      <path value="ServiceRequest.orderDetail" />
+      <sliceName value="procedureMethod" />
+      <short value="ProcedureMethod" />
+      <definition value="The method or technique that will be used to perform the procedure, e.g. approach, lavage, pressuring, ets." />
+      <alias value="VerrichtingMethode" />
+      <binding>
+        <strength value="required" />
+        <valueSet value="http://decor.nictiz.nl/fhir/ValueSet/2.16.840.1.113883.2.4.3.11.60.40.2.14.1.4--20200901000000" />
+      </binding>
+      <mapping>
+        <identity value="zib-procedure-v5.2-2020EN" />
+        <map value="NL-CM:14.1.12" />
+        <comment value="ProcedureMethod" />
+      </mapping>
+    </element>
+    <element id="ServiceRequest.occurrence[x]">
+      <path value="ServiceRequest.occurrence[x]" />
+      <short value="ProcedureStartDate / ProcedureEndDate" />
+      <definition value="* The (desired) start date (and if possible start time) of the procedure. A ‘vague’ date, such as only the year, is permitted. &#xD;&#xA;* The element offers the option to indicate the start of the period of a series of related procedures. &#xD;&#xA;* The end date (and if possible end time) of the procedure. A ‘vague’ date, such as only the year, is permitted. &#xD;&#xA;* The element offers the option to indicate the end of the period of a series of related procedures. The end date element is only used for a procedures that takes some time and is then always applied. If the procedure still continues, the value is left empty. For instantaneous or very short lasting procedures the element is omitted." />
+      <comment value="The `.occurrencePeriod.start` and `.occurrencePeriod.end` are used for ProcedureStartDate and ProcedureEndDate if the zib Procedure concerns a period. For instantaneous or very short lasting procedures the `.occurrenceDateTime` is used." />
+      <alias value="VerrichtingStartDatum" />
+      <alias value="VerrichtingEindDatum" />
+      <mapping>
+        <identity value="zib-procedure-v5.2-2020EN" />
+        <map value="NL-CM:14.1.2" />
+        <comment value="ProcedureStartDate" />
+      </mapping>
+      <mapping>
+        <identity value="zib-procedure-v5.2-2020EN" />
+        <map value="NL-CM:14.1.3" />
+        <comment value="ProcedureEndDate" />
+      </mapping>
+    </element>

```

Comment:
Don't know if it is possible to slice for occurrenceDateTime and occurrencePeriod, or is it possible to specify that occurrenceTiming should not be used in some way?

Reply:
Well, they can be sliced, but then you will have double mappings. We shouldn't restrict `.occurrenceTiming` too. This relates to #52

Reply:
Ha no I didn't mean restricting occurenceTiming, I only meant making the mapping explicitly only about occurrenceDateTime and occurrencePeriod. But if that means 2 slices and 2 mappings instead of both DateTime and Period in 1 mapping, then this is better I guess

Reply:
Ok, lets leave this for now. If we get better insights in #53, we can adjust it accordingly and do it consequent for the other zib-profiles too.